tests/parental_consent.spec.ts

```diff
@@ -56,7 +56,7 @@ async function given_the_app_is_setup() {
 }
 
 async function when_i_go_to_the_consent_start_page() {
-  await p.goto("/sessions/1/consents/start");
+  await p.goto("/sessions/2/consents/start");
```

Comment:
2nd campaign is flu in the sample data which is what we're testing for here (we check that the first question is what we expect)